sdk/core/core-amqp/package.json

```diff
@@ -43,7 +43,7 @@
     "clean": "rimraf dist dist-* temp types coverage coverage-browser .nyc_output *.tgz *.log test*.xml",
     "execute:samples": "echo skipped",
     "extract-api": "tsc -p . && api-extractor run --local",
-    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
```

Comment:
We don't want to format the samples?

cc @JonathanCrd In case this needs to be standardized across packages

Reply:
there is no samples in this package.